src/pre-engagement-form/form-components/types.ts

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
I'm not sure if this is considered some "generic naming" and goes against the new standard for TS, or if we are fine with the `types.ts` files for this use cases (it's very well scoped in my opinion, this can only contain type definitions for the `form-components` module :P). Thoughts @stephenhand ?